service/src/test/resources/submitted-form-pdf-content.txt

```diff
@@ -11,8 +11,8 @@ Contested interim care orderHearing type
 Reason: Baby will be discharged from hospital on 21 June
```

Comment:
I wished value of that file be greater. At the moment it only detects accidental changes :(